src/test/javascript/spec/component/file-upload-assessment/file-upload-assessment-dashboard.component.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { TranslateService } from '@ngx-translate/core';
+import { ArtemisTestModule } from '../../test.module';
+import { ActivatedRoute, convertToParamMap } from '@angular/router';
+import { of } from 'rxjs';
+import { JhiLanguageHelper } from 'app/core/language/language.helper';
+import { AccountService } from 'app/core/auth/account.service';
+import { MockAccountService } from '../../helpers/mocks/service/mock-account.service';
+import { MockSyncStorage } from '../../helpers/mocks/service/mock-sync-storage.service';
+import { MockTranslateService } from '../../helpers/mocks/service/mock-translate.service';
+import { LocalStorageService, SessionStorageService } from 'ngx-webstorage';
+import { ExerciseType } from 'app/entities/exercise.model';
+import { SortService } from 'app/shared/service/sort.service';
+import { FileUploadAssessmentDashboardComponent } from 'app/exercises/file-upload/assess/file-upload-assessment-dashboard.component';
+import { FileUploadSubmissionService } from 'app/exercises/file-upload/participate/file-upload-submission.service';
+import { FileUploadAssessmentsService } from 'app/exercises/file-upload/assess/file-upload-assessment.service';
+
+const course = { id: 1 };
+const testRoute = ({ snapshot: { paramMap: convertToParamMap({ courseId: course.id, exerciseId: 22 }) } } as any) as ActivatedRoute;
+const fileuploadExercise = { id: 22, course, type: ExerciseType.FILE_UPLOAD, studentAssignedTeamIdComputed: true };
+const fileUploadSubmission = { id: 25, submitted: true, result: { id: 10, assessor: { id: 20, guidedTourSettings: [] } } };
+
+describe('FileUploadAssessmentDashboardComponent', () => {
+    let component: FileUploadAssessmentDashboardComponent;
+    let fixture: ComponentFixture<FileUploadAssessmentDashboardComponent>;
+    let fileUploadSubmissionService: FileUploadSubmissionService;
+    let fileUploadAssessmentsService: FileUploadAssessmentsService;
+    let sortService: SortService;
+    let mockAuth: MockAccountService;
+
+    beforeEach(fakeAsync(() => {
+        TestBed.configureTestingModule({
+            imports: [ArtemisTestModule],
+            declarations: [FileUploadAssessmentDashboardComponent],
+            providers: [
+                JhiLanguageHelper,
+                { provide: ActivatedRoute, useValue: testRoute },
+                { provide: LocalStorageService, useClass: MockSyncStorage },
+                { provide: SessionStorageService, useClass: MockSyncStorage },
+                { provide: TranslateService, useClass: MockTranslateService },
+            ],
+        })
+            .overrideTemplate(FileUploadAssessmentDashboardComponent, '')
+            .compileComponents()
+
```

Comment:
The new line here looks a bit weird to me, but I don't know what the style policy for this is.